### ISSUE TITLE
Save all mess up code

### ISSUE THREAD

**Issue by klauszhang** [Lonefy/vscode-JS-CSS-HTML-formatter#39]
- VSCode Version:1.8.1
- OS Version: windows 10 Pro 10.0.14393 Build 14393

Steps to Reproduce:

1.  change more than 2 files without save,
2.  save all

then application start to become crazy... copy all content from one file to another. mess up everything, complaining the file on the disk actually newer that the one opened..
I tried to disable all extensions and it still behave funny. PLEASE FIX THIS!

I lost some unstaged code because of this.

**Comment by PooperPig** [comment]
VSCode Version: 1.8.1
OS Version: MacOS 10.12.2
Steps to Reproduce:

Open two existing files in the editor
Make a small change to one, do not save
Move to the tab of the unchanged file
Hit cmd-alt-s
The files are saved, and the contents of the changed file pasted into the unchanged file.
See attached screen capture for example on newly created project
[VSCodeSaveAllError.mov.zip](https://github.com/Lonefy/vscode-JS-CSS-HTML-formatter/files/686033/VSCodeSaveAllError.mov.zip)

**Comment by WhatFreshHellIsThis** [comment]
Wow, this **still** isn't fixed?!?!?!?  This is a horrible issue and was reported a *long* time ago.  I'm going to uninstall this extension now and I advise no one to use it until (if ever) this issue is fixed, it has led several times to loss of work now and having to retrieve from git damaged files.

**Comment by SathishRaju** [comment]
I agree on the above comment. It messes up the entire code and i have to find a way to restore it. Please fix this!
